src/Files/Views/Pages/PropertiesGeneral.xaml.cs

```diff
@@ -108,5 +109,10 @@ public override async Task<bool> SaveChangesAsync(ListedItem item)
         public override void Dispose()
         {
         }
+
+        private async void DiskCleanupButton_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+            await Launcher.LaunchUriAsync(new Uri("ms-settings:storagesense"));
```

Comment:
Could you include the same logic as found in [DrivesWidget.xaml.cs](https://github.com/files-community/Files/blob/main/src/Files/UserControls/Widgets/DrivesWidget.xaml.cs#L202)?

Reply:
Is there a way to put the code one location to make it easier to reuse?